libs/clients/signature-collection/src/lib/types/list.dto.ts

```diff
@@ -0,0 +1,41 @@
+import { Area } from './area.dto'
+import { UserBase } from './user.dto'
+import { MedmaelalistiDTO } from '../../../gen/fetch'
+
+export interface List {
+  id: string
+  title: string
+  owner: UserBase
+  area: Area
+  active: boolean
+  startTime: Date
+  endTime: Date
+  collectionId?: string
+  collectors?: UserBase[]
+  numberOfSignatures: number
+  link?: string
+}
+
+export function mapList(list: MedmaelalistiDTO): List {
+  // TODO: Aggregate owners
+  // TODO: Status of open list, closed, extended
+  return {
+    id: list.id?.toString() ?? '',
+    collectionId: list.medmaelasofnun?.id?.toString() ?? '',
+    title: list.listiNafn ?? '',
+    startTime: list.medmaelasofnun?.sofnunStart ?? new Date(),
+    endTime: list.medmaelasofnun?.sofnunEnd ?? new Date(),
+    area: {
+      id: list.svaedi?.id?.toString() ?? '',
+      name: list.svaedi?.nafn?.toString() ?? '',
+      min: list.svaedi?.fjoldi ?? 999,
```

Comment:
Same suggestion
```suggestion
      min: list.svaedi?.fjoldi ?? Number.MAX_SAFE_INTEGER,
```

Reply:
Changing no 0, this should not happen but the generated client is returning everything as optional :/